fastlane-plugin-revenuecat.gemspec

```diff
@@ -0,0 +1,36 @@
+lib = File.expand_path("lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'fastlane/plugin/revenuecat/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'fastlane-plugin-revenuecat'
```

Comment:
Thoughts on a potential rename to something like `fastlane-plugin-revenuecat-internal` along with a repo rename? 😅 

Was just thinking that we _may_ want to keep the `fastlane-plugin-revenuecat` namespace and repo available for if/when we ever create a _fastlane_ plugin for our customers to use 🤷‍♂️

Reply:
That makes a lot of sense 👍 Will do the rename in a bit.

And what do you think about publishing this plugin in RubyGems? I honestly thought it might be a better idea to have the SDKs just point to github, since this is only going to be internal, but if you think we would benefit in any way from publishing I can work on that